docs/sql-data-sources-text.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: global
+title: Text Files
+displayTitle: Text Files
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+Spark SQL provides `spark.read().text("file_name")` to read a file or directory of text files into a Spark DataFrame, and `dataframe.write().text("path")` to write to a text file. When reading a text file, each line becomes each row that has string "value" column by default. The line separator can be changed as shown in the example below. When specifying a directory as a file path, make sure that the files included in the directory do not contain a format that is inappropriate for reading text, such as ORC or Parquet. The `option()` function can be used to customize the behavior of reading or writing, such as controlling behavior of the line separator, compression, and so on.
```

Comment:
"When specifying a directory as a file path, make sure that the files included in the directory do not contain a format that is inappropriate for reading text, such as ORC or Parquet" I think this is too much to know or document.

Reply:
Okay, seems like I'd better to just remove these kinda vervose.